Gemfile

```diff
@@ -151,7 +151,7 @@ group :development do
   # listen higher than 3.1.1 require Ruby version >= 2.2.3 (we're currently on
   # 2.2.2). Restrict the version of `listen` to prevent `guard-rspec`
   # introducing an incompatible dependency:
-  gem 'listen', '~> 3.0.5', '<= 3.1.1'
+  gem 'listen', '>= 3.0.5', '< 3.2'
```

Comment:
I have this error when running `bundle`:
```
Gem::InstallError: ruby_dep requires Ruby version >= 2.2.5, ~> 2.2.
An error occurred while installing ruby_dep (1.5.0), and Bundler cannot continue.
Make sure that `gem install ruby_dep -v '1.5.0'` succeeds before bundling.
```
It works when changing this line with:
`gem 'listen', '~> 3.0.8'`